deepcell_dot_org/README.md

```diff
@@ -1,10 +1,57 @@
 # Introduction to DeepCell.org
 
-Noah to add.
+Deepcell.org is a web-based interface to access our pre-trained deep learning models. The website allows you to easily upload example images, run them on our available models, and download the results without requiring any local installation. The backend is managed by the [kiosk](https://github.com/vanvalenlab/kiosk-console), which provides an efficient and scalable way to analyze large volumes of data using cloud computing.
 
 ## Table of Contents
 
-Ready-made predictions on real data: DeepCell.org
+* [Formatting data for web-based models](README.md/#formatting-data-for-web-based-models)
+* [Generating predictions with DeepCell.org](README.md/#generating-predictions-with-deepcellorg)
+* [Plugging in to DeepCell.org with ImageJ](README.md/#plugging-in-to-deepcellorg-with-imagej)
 
-* [Introduction to DeepCell.org](./deepcell_dot_org)
-* [Plugging in to DeepCell.org with ImageJ](./deepcell_dot_org)
+## Formatting data for web-based models
+
+#### Multiplex Model
+
+The multiplex model performs whole-cell segmentation of tissue imaging data. The input to the model is two-channel images. The first channel must be a nuclear channel (such as DAPI). The second channel must be a membrane or cytoplasmic channel (such as E-Cadherin).
+
+#### Nuclear Segmentation Model
+
+TODO: do we describe both nuclear and cytoplasm model? Only one? Keep this blank?
```

Comment:
I don't know where we landed on including the cytoplasm model, but I'm leaning towards only the nuclear segmentation.

Maybe something simple like:

> The nuclear segmentation model takes single-channel nuclear data (such as DAPI).

Reply:
Let's focus on nuclear only for now